src/main/scala/tech/cryptonomic/conseil/routes/Service.scala

```diff
@@ -0,0 +1,42 @@
+package tech.cryptonomic.conseil.routes
+
+import akka.http.scaladsl.marshalling.{PredefinedToEntityMarshallers, ToEntityMarshaller}
+import akka.http.scaladsl.model.MediaTypes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Route, StandardRoute}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import tech.cryptonomic.conseil.tezos.ServiceOperations
+import tech.cryptonomic.conseil.util.JsonUtil._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object Service {
+  def apply(config: Config)(implicit apiExecutionContext: ExecutionContext): Service = new Service(config)
+}
+
+class Service(config: Config)(implicit apiExecutionContext: ExecutionContext) extends LazyLogging {
+  val route: Route =
+    get {
+      pathPrefix("networks") {
+        pathEnd {
+          complete(toJson(ServiceOperations.getNetworks(config)))
+        } ~ pathPrefix(Segment) { network =>
+          pathPrefix("entities") {
+            pathEnd {
+              completeWithJson(ServiceOperations.getEntities(network))
+            }
+          }
+        }
+      }
+    }
+
+  implicit private val jsonMarshaller: ToEntityMarshaller[JsonString] =
```

Comment:
Should we factor out these common methods in an external trait or object that can be mixed-in or imported in both routes (this and `tezos`)?